src/app/(pages)/home/_components/OurValuesBlock.tsx

```diff
@@ -78,7 +78,8 @@ function Value({ data }: { data: ValueDataType }) {
       <div
         className={cn(
           "hidden md:flex top-1/2 w-[100px]  md:w-[180px] lg:w-[220px] aspect-square transform absolute -translate-y-1/2  justify-center items-center z-4 right-2 translate-x-full",
-          data.flip && "-left-2 -translate-x-full"
+          data.flip && "-left-2 -translate-x-full",
+          data.title === "Mission First." && "w-[300px] md:w-[300px] lg:w-[300px] md:translate-x-[260px] lg:translate-x-[260px]"
```

Comment:
lets take out this line and adjust the actual mission.svg file in ../_assets/ourValues/
i noticed the svg doesn't look the same on our website vs on the design so i just copied the svg from there and pasted it into the mission.svg file and that seemed to help a lot
<img width="549" alt="image" src="https://github.com/user-attachments/assets/8d08a1a6-bb60-4187-bc7d-da67856aeaba">